Flutter/cartesian-charts/axis-types.md

```diff
@@ -158,7 +213,13 @@ When the value of [`rangePadding`](https://pub.dev/documentation/syncfusion_flut
                         primaryYAxis: NumericAxis(
                             // Additional range padding is applied to y axis
                             rangePadding: ChartRangePadding.additional
-                        )  
+                        ),
+                        series: <ChartSeries<ChartData, int>>[
```

Comment:
why do we need to add the series without data?

Reply:
In `### Customizing range` datasource and below data's are same, so I didn't add.